Source/Core/Credit.js

```diff
@@ -2,61 +2,42 @@ define([
     './defaultValue',
     './defined',
     './defineProperties',
-    './DeveloperError'
+    './deprecationWarning',
+    './DeveloperError',
+    '../ThirdParty/xss'
 ], function(
     defaultValue,
     defined,
     defineProperties,
-    DeveloperError) {
+    deprecationWarning,
+    DeveloperError,
+    xss) {
     'use strict';
 
     var nextCreditId = 0;
     var creditToId = {};
 
-    function getElement(credit) {
-        var element = document.createElement('span');
-        var text = credit.text;
-        var link = credit.link;
-        var a;
-        if (credit._hasImage) {
-            var content = document.createElement('img');
-            content.src = credit.imageUrl;
-            if (defined(text)) {
-                content.alt = text;
-                content.title = text;
-            }
-            if (credit._hasLink) {
-                a = document.createElement('a');
-                a.appendChild(content);
-                a.href = link;
-                a.target = '_blank';
-                element.appendChild(a);
-            } else {
-                element.appendChild(content);
-            }
-            element.className = 'cesium-credit-image';
-        } else {
-            if (credit._hasLink) {
-                a = document.createElement('a');
-                a.textContent = text;
-                a.href = link;
-                a.target = '_blank';
-                element.appendChild(a);
-            } else {
-                element.textContent = text;
-            }
-            element.className = 'cesium-credit-text';
+    function createDomNode(html) {
+        var div = document.createElement('span');
+        div.innerHTML = html;
+
+        if (div.children.length === 1) {
```

Comment:
This check is bad, I'm not sure you need it at all but if you're trying to avoid an extra wrapper then it should be `childNodes` not `children`

Reply:
You can test with `This entire credit should be <b>visible</b>`